test/adapters/level_zero/CMakeLists.txt

```diff
@@ -3,89 +3,108 @@
 # See LICENSE.TXT
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-if(UR_BUILD_ADAPTER_L0)
+if(NOT WIN32 AND NOT UR_STATIC_ADAPTER_L0)
```

Comment:
this is terrible, and we should remove the test that requires this.

Reply:
I know... Right now we have 2 tests that use this I belive. I was thinking that perhaps until we have a better way we could just move the checks outside of the binary to some script - that is, just print some kind of markers in the test binary and redirect it along with ZE traces to a file and then do matching in python. Anyway, that's something for a separate PR.

Reply:
Done: https://github.com/oneapi-src/unified-runtime/pull/2431